### ISSUE TITLE
Missing alt text on director headshot

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#1314]
**URL**
https://alpha.austin.gov/police-oversight
https://alpha.austin.gov/es/police-oversight

**Screenshots**
![image.png](https://images.zenhubusercontent.com/5a31a50b8a75884b908b0ea6/5b533cb7-39b6-4431-b36f-6e3d96e92eb7)
